src/views/Recorder/Recorder.tsx

```diff
@@ -205,47 +194,33 @@ export function Recorder() {
         )
       }
     >
-      <Allotment defaultSizes={[1, 1]}>
-        <Allotment.Pane minSize={200}>
-          <Flex direction="column" height="100%">
-            <div css={{ flexGrow: 0, minHeight: 0 }}>
-              <RequestsSection
-                proxyData={debouncedProxyData}
-                noDataElement={noDataElement}
-                selectedRequestId={selectedRequest?.id}
-                autoScroll
-                groups={groups}
-                activeGroup={group?.id}
-                onSelectRequest={setSelectedRequest}
-                onUpdateGroup={handleUpdateGroup}
-                resetProxyData={handleResetRecording}
-              />
-            </div>
-            {recorderState === 'recording' && (
-              <Box width="200px" p="2">
-                <ButtonWithTooltip
-                  size="2"
-                  variant="ghost"
-                  ml="2"
-                  onClick={() => handleCreateGroup(`Group ${groups.length}`)}
-                  tooltip="Groups are used to organize specific steps in your recording. After you create a group, any further requests will be added to it."
-                >
-                  <PlusCircledIcon />
-                  Create group
-                </ButtonWithTooltip>
-              </Box>
-            )}
-          </Flex>
-        </Allotment.Pane>
-        {selectedRequest !== null && (
-          <Allotment.Pane minSize={300}>
-            <Details
-              selectedRequest={selectedRequest}
-              onSelectRequest={setSelectedRequest}
-            />
-          </Allotment.Pane>
+      {recorderState === 'idle' && (
+        <EmptyState isLoading={isLoading} onStart={handleStartRecording} />
+      )}
+
+      {recorderState !== 'idle' && settings?.recorder.enableBrowserRecorder && (
+        <RecordingInspector
+          recorderState={recorderState}
+          groups={groups}
+          requests={debouncedProxyData}
+          browserEvents={browserEvents}
+          onCreateGroup={handleCreateGroup}
+          onUpdateGroup={handleUpdateGroup}
+          onResetRecording={handleResetRecording}
+        />
+      )}
+
+      {recorderState !== 'idle' &&
+        !settings?.recorder.enableBrowserRecorder && (
+          <RequestLog
+            recorderState={recorderState}
+            groups={groups}
+            requests={debouncedProxyData}
+            onUpdateGroup={handleUpdateGroup}
+            onResetRecording={handleResetRecording}
+            onCreateGroup={handleCreateGroup}
+          />
```

Comment:
I'm trying to keep the normal UI when you disable the browser feature. Maybe it makes more sense to always show the tabbed interface and disable/hide the "Browser Events" tab?